androidApp/src/main/java/com/projectnewm/ui/theme/Type.kt

```diff
@@ -0,0 +1,28 @@
+package com.projectnewm.ui.theme
+
+import androidx.compose.material.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+// Set of Material typography styles to start with
+val Typography = Typography(
+    body1 = TextStyle(
+        fontFamily = FontFamily.Default,
```

Comment:
Where is it getting the default from? XML theme?

Reply:
Compose seems to have default values based on the Material theme